src/SWP/Bundle/CoreBundle/Controller/WidgetController.php

```diff
@@ -84,6 +84,11 @@ public function getAction(Request $request, $id)
     /**
      * Create new widget.
      *
+     * Note:
+     *
+     *     Widget Type can be widget ID or his class (string).
+     *     Example: "SWP\\Bundle\\CoreBundle\\Widget\\ContentListWidget"
```

Comment:
This fix doesn't change anything because type of the widget will be still a class name `SWP\\Bundle\\CoreBundle\\Widget\\ContentListWidget` which is not readable for end user. A mapping should introduced so for example `SWP\\Bundle\\CoreBundle\\Widget\\ContentListWidget` would map to `list` type which is more user friendly and internally we could use class namespace. So in request to create new form you would type `list` widget type. At least ticket is about it afaik.